_posts/2016/2016-04-21-jenadevs-docker-party.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "First Jena Devlopers Meetup: Docker Party!"
+date: "2016-04-11 13:37:42"
+icon: cubes
+categories: events
+authors: ["Benjamin and Oliver Z."]
+---
+
+On the first Thursday of April the newly founded [Softwerkskammer Jena](https://www.softwerkskammer.org/groups/jena) had its first [Developers Meetup](www.meetup.com/jenadevs).
+ePages took the opportunity to be the event host and provide not just conference rooms and serveral workstation laptops but also free pizza, snacks, [Club-Mate](https://de.wikipedia.org/wiki/Club-Mate) and some other refreshments for everyone.
+Despite the short notice over 30 attendees from a dozen of companies from the area of Jena, university students and even some freelancers followed the invitation to the Docker Party.
+They joined the orga team of 8 developers for 4 hours of coding fun from 6 pm until nearly 10 pm with an open end at the local pub, where the devoted discussions lasted until 2 am in the morning.
+
+## Agenda
+
+At the beginning there was a short period of spare time for meet and greet as well as a brief introduction about the [Softwerkskammer](softwerkskammer.org) – the german part of the [Software Craftsmanship Community](http://manifesto.softwarecraftsmanship.org) – and its current distribution with some helpful contact information like the [@jenadevs](https://twitter.com/jenadevs) account on Twitter.
+
+Then the Docker Party, which celebrated the [3rd birthday of Docker](https://www.docker.com/community/docker-birthday-3), was launched! 
+It consisted of an one hour talk including a demo to introduce the technological foundation and nearly three hours of different workshops for every skill level:
```

Comment:
The agenda included a one hour talk along with a demo to introduce the technological foundation as well as different workshops for every skill level: